man/systemd.network.xml

```diff
@@ -1247,7 +1247,15 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
       <varlistentry>
         <term><varname>TypeOfService=</varname></term>
         <listitem>
-          <para>Takes a number between 0 and 255 that specifies the type of service to match.</para>
+          <para>
+            This specifies the Type of Service (ToS) field of packets to match;
+            it takes an unsigned integer in the range 0…255.
```

Comment:
@keszybz suggests `an unsigned integer` -> `a number`, though I do not have any preference about that.

Reply:
There are 27 occurrences of "unsigned integer" in the docs today. If we're going to change them, let's not do it ad hoc and change them all at once if everybody agrees. (Or maybe go through all 27 and consider each one.)

Reply:
OK. Let's change that later consistently.